misc/librenms-irc.service

```diff
@@ -0,0 +1,12 @@
+# /etc/systemd/system/librenms-irc.service 
+
+librenms-irc.service
+[Unit]
+Description=Librenms IRC bot
+
+[Service]
+ExecStart=/opt/librenms/irc.php
```

Comment:
Just my 2 cents, on Debian based systems central config directory is /etc/default. Also I'd like to prepare sysv init scripts for any daemons as well, so are there any more daemons?